### DISCUSSION TITLE
Themes and component libraries

### DISCUSSION THREAD

**Discussion by richburdon** [vercel/next.js#12876] [Help]
Hi everyone, I've built a couple of simple nextjs sites, but relatively new.

Are there popular GH repos (or npm components) for clean react components or one page sites with themes? I see a bunch of "templates" on sites like themeforest.net, but they are very heavy and "all or nothing". I want to build a site using small reusable components. I know this isn't really a nextjs issue (you can use any react components), but wondering if there's a library of components that would be generally useful for building themed sites that feel connected.

Thanks

**Comment by sambhav-gore** [comment]
how about https://material-ui.com/ ?

there is Also AntD - https://ant.design/

there are few more UI libraries available on web. but those two are most popular IMO.

also SprintUI is in VIP presale right now  https://sprintui.com/ (it can be used with react with some work )
